README.md

```diff
@@ -277,6 +277,20 @@ $ go-licenses check \
 Note that dependencies from the ignored packages are still resolved and checked.
 This flag makes effect to `check`, `report` and `save` commands.
 
+### Include testing packages
+
+Use the `--include_tests` global flag to include packages only imported by testing code.
+For example, those packages could be testing libraries/frameworks.
+Usually you would want to add this flag because if you're already analyzing the licenses of your dependencies,
+likely those packages also need to be taken into account.
```

Comment:
probably omit this?

you described what effect this flag has, I think it is reasonable to let users decide whether they need the flag.

I would use the flag when running check, because all deps must be allowed.

when saving report for a binary, I won't use it, because testing libraries are not included.

Reply:
Good point, updated.